src/api/controllers/PostController.ts

```diff
@@ -27,7 +27,7 @@ export class PostController {
   }
 
   @Get('id/:id/')
-  async getPostById(@Params() params: UuidParam): Promise<GetPostResponse  | undefined> {
+  async getPostById(@Params() params: UuidParam): Promise<GetPostResponse> {
```

Comment:
I think it was previously undefined as well in case there was no post given the id. But now we have proper error throwing/handling so this is unnecessary, must've just forgotten to delete the undefined from this endpoint specifically